partnerships/community_success/hub-activity.md

```diff
@@ -0,0 +1,311 @@
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
++++ {"editable": true, "slideshow": {"slide_type": ""}, "user_expressions": []}
+
+# Track Hub Usage with Grafana
+
++++ {"user_expressions": []}
+
+## Overview
+
+Grafana is an open source analytics and interactive visualization web application. Prometheus is an open-source monitoring and alerting platform that collects and stores metrics as time-series data, which feeds into Grafana as a data source.
+
+Grafana dashboard deployments for 2i2c hubs (k8s+JupyterHub) follow the templates outlined in the GitHub repository [jupyterhub/grafana-dashboards](https://github.com/jupyterhub/grafana-dashboards). Note that Prometheus data is retained for 1 year on 2i2c hubs.
+
++++ {"user_expressions": []}
+
+## Prerequisites
+
+### Create a Grafana service account and generate a token
+
+See [Grafana docs – Service Accounts](https://grafana.com/docs/grafana/latest/administration/service-accounts/) for more details.
+
+1. Navigate to the [2i2c Grafana](https://grafana.pilot.2i2c.cloud) website at `https://grafana.pilot.2i2c.cloud`.
+1. Open the {octicon}`three-bars` Menu and click on *{octicon}`gear` Administration > Users and access > Service accounts*.
+1. Click on the *Add service account* button on the top-right.
+1. Choose a descriptive *Display name*, e.g. `username-local-prometheus-access` and leave the role as *Viewer*. Click the *Create* button to confirm.
+1. You will see a new page with the details of the service account you have created. In the section *Tokens*, click the *Add service account token* button to generate a token to authenticate with the Grafana API.
+1. Choose a descriptive name for the token and then set a token expiry date. We recommend 1 month from now.
+1. Click the *Generate token button* to confirm.
+1. **Important:** Copy the token and keep a copy somewhere safe. You will not be able to see it again. Losing a token requires creating a new one.
+
+### Access the Grafana Token locally in a .env file 
+
+Access your Grafana token in a local development environment by storing it in a `.env` file.
+
+:::{caution}
+Keep your Grafana token secure and do not upload this to a Git repo. Add `.env` to your `.gitignore` file.
+:::
+
+1. Create a new `.env` file in your working directory.
+1. Edit the `.env` file to include
+
+   ```bash
+   GRAFANA_TOKEN=<paste your Grafana token here>
+   ```
+1. Save and close.
+
+### Access the Grafana Token in GitHub actions
+
+Add your Grafana token as a GitHub repository secret to be used in GitHub actions when you build and publish online.
+
+:::{note}
+This section describes how to create a secret for an individual repository. To create a secret for the 2i2c organization, see the [GitHub Docs – Using secrets in GitHub actions](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-an-organization).
+:::
+
+1. Navigate to your repository online on GitHub.
```

Comment:
This feels like it could be a standalone section on "how to add secrets to a GitHub repository" that we just link out to from here, since that's a broadly useful action. What do you think?

By the way, this is an example of something where we could re-use content via `{embed}` directives when we switch to mystmd.org (https://mystmd.org/guide/embed#embedding-from-external-projects)